api/models/User.js

```diff
@@ -547,7 +547,8 @@ module.exports = bookshelf.Model.extend(merge({
     } else {
       q = User.where({ id })
     }
-    return q.where('users.active', active).fetch(options)
+    if (active) return q.where('users.active', active).fetch(options)
```

Comment:
why do we need this? this means we couldnt pass in active: false to search for inactive users

Reply:
I added the `active` parameter earlier in this work, so that the reactivate mutation would work. I made it only search `active = false` but it doesn't need to be that specific. The original behavior was searching amongst `active = true`, which I have preserved as the default behavior.

The changes I have added need to search both `active = true` and `active = false`, since it needs to return a user whether they are active or not

Reply:
its just weird to pass in active = false and have that mean to search for both active and non-active users, at least comment it